src/options/modules/ColorSchemeModeHelper.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * // does not work in Firefox currently due to https://bugzilla.mozilla.org/show_bug.cgi?id=1595037
```

Comment:
Also please explain the function's objective etc, as it is best practise for JSDoc.